recipes/harfbuzz/all/patches/source_subfolder.patch

```diff
@@ -2,10 +2,7 @@ diff --git a/source_subfolder/CMakeLists.txt b/source_subfolder_patched/CMakeLis
 index 2a8fd8b..2b7056c 100644
 --- a/source_subfolder/CMakeLists.txt
 +++ b/source_subfolder_patched/CMakeLists.txt
-@@ -234,19 +234,19 @@ if (HB_HAVE_ICU)
-   add_definitions(-DHAVE_ICU)
```

Comment:
Isn't this define needed to optionally include/use icu inside the sources?

Reply:
I did not remove it from the file, but just from the patch, because the comment below changed in 2.7.4 sources

Reply:
Oh sorry.
I didn't understand the complete context.
I was looking at this from my mobile phone.

Reply:
There's only one patch for 2.7.2 and 2.7.4, just to make sure this change is also needed for lower versions?

Reply:
I don't get your question. this patch is applied to all versions.

Reply:
perfect, you answered my question =D